src/main/resources/assets/susy/lang/en_us.lang

```diff
@@ -759,6 +759,10 @@ gregtech.machine.fluid_hatch.export_9x.hv.name=HV Nonuple Output Hatch
 gregtech.machine.dumping_hatch=Dumping Hatch
 gregtech.machine.dumping_hatch.lv.name=Dumping Hatch
 
+# Distillation tower explanations
+gregtech.machine.ordered_dt.tooltip.1=Fluid input and output hatches are on the same layers
```

Comment:
rename this to "Fluid input and output hatches _can be_ placed in the same layer.

Reply:
I was considering this, but since you basically have to do that if you don't want to void items, I decided to pick "are" instead.

Reply:
this will get confusing for recipes with 2 inputs and 3 outputs tho.